modules/test/playwright/tests/account-admin-web/accountGroups.spec.ts

```diff
@@ -66,3 +66,65 @@ test('LPD-25948 Can search for account group', async ({
 		0
 	);
 });
+
+test('LPD-28159 Account groups are displayed in account details page', async ({
+	accountsPage,
+	apiHelpers,
+	page,
+}) => {
+	const account = await apiHelpers.headlessAdminUser.postAccount({
+		name: getRandomString(),
+		type: 'business',
+	});
+
+	apiHelpers.data.push({id: account.id, type: 'account'});
+
+	const accountGroup1 = await apiHelpers.headlessAdminUser.postAccountGroup({
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: accountGroup1.id, type: 'accountGroup'});
+
+	const accountGroup2 = await apiHelpers.headlessAdminUser.postAccountGroup({
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: accountGroup2.id, type: 'accountGroup'});
+
+	const accountGroup3 = await apiHelpers.headlessAdminUser.postAccountGroup({
+		name: getRandomString(),
+	});
+
+	apiHelpers.data.push({id: accountGroup3.id, type: 'accountGroup'});
+
+	await apiHelpers.headlessAdminUser.assignAccountToAccountGroup(
+		account.externalReferenceCode,
+		accountGroup1.externalReferenceCode
+	);
+
+	await apiHelpers.headlessAdminUser.assignAccountToAccountGroup(
+		account.externalReferenceCode,
+		accountGroup2.externalReferenceCode
+	);
+
+	await apiHelpers.headlessAdminUser.assignAccountToAccountGroup(
+		account.externalReferenceCode,
+		accountGroup3.externalReferenceCode
+	);
+
+	await accountsPage.goto();
+	await (await accountsPage.accountsTableRowLink(account.name)).click();
+	await accountsPage.accountGroupsTab.click();
+
+	await expect(
+		page.getByText(accountGroup1.name, {exact: true})
```

Comment:
can you add a dynamic locator (so a function - we have already example) in the page and in the test call it passing the name? I don't like to have the locators (so the .getBy***) in the tests - and you can call the same dynamic locator three times with the three different account groups